tests/e2e/specs/aotf.js

```diff
@@ -19,22 +19,110 @@ import {
   mutationStatus
 } from '@/utils/aotf'
 
+const TYPES = []
+const MUTATIONS = [
+  {
+    name: 'workflowMutation',
+    _title: 'Workflow Mutation',
+    description: `
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+      Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+      enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+      aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit
+      in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+      Excepteur sint occaecat cupidatat non proident, sunt in culpa qui
+      officia deserunt mollit anim id est laborum.
+    `,
+    args: [
+      {
+        name: 'workflow',
+        type: {
+          name: 'WorkflowID',
+          kind: null
+        }
+      }
+    ]
+  },
+  {
+    name: 'cycleMutation',
+    _title: 'Cycle Mutation',
+    description: 'cycle',
+    args: [
+      {
+        name: 'workflow',
+        type: {
+          name: 'WorkflowID',
+          kind: null
+        }
+      },
+      {
+        name: 'cycle',
+        type: {
+          name: 'CyclePoint',
+          kind: null
+        }
+      }
+    ]
+  },
+  {
+    name: 'namespaceMutation',
+    _title: 'Namespace Mutation',
+    description: 'namespace',
+    args: [
+      {
+        name: 'workflow',
+        type: {
+          name: 'WorkflowID',
+          kind: null
+        }
+      },
+      {
+        name: 'namespace',
+        type: {
+          name: 'NamespaceName',
+          kind: null
+        }
+      }
+    ]
+  },
+  {
+    name: 'jobMutation',
+    _title: 'Job Mutation',
+    description: 'job',
+    args: [
+      {
+        name: 'workflow',
+        type: {
+          name: 'WorkflowID',
+          kind: null
+        }
+      },
+      {
+        name: 'job',
+        type: {
+          name: 'JobID',
+          kind: null
+        }
+      }
+    ]
+  }
+]
+
```

Comment:
This was in the `MockWorkflowService`, but was used only for this e2e test. We can perhaps make it permanent in the offline mode, so that we can play with mutations. However, it would require more changes (that I started doing and gave up as it would bloat this PR) to the WebSockets code added here first.